v1.1-beta/matrixx-digital-commerce/matrixx-chf-standalone/cnf-testsuite.yml

```diff
@@ -0,0 +1,18 @@
+---
+helm_directory: "chf-standalone"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-784450778.yaml\
```

Comment:
Please make this generic so that others can use it.  For example:
```--values ~/.mtx/konstruxx/working/values-784450778.yaml```
or
```--values values-784450778.yaml" assuming they will run this in the same directory as the cnf-testsuite.yml```